src/main/java/bookstore/mapper/BookMapper.java

```diff
@@ -1,22 +1,23 @@
 package bookstore.mapper;
 
-import bookstore.dto.BookRequestDto;
-import bookstore.dto.BookResponseDto;
+import bookstore.dto.book.BookRequestDto;
+import bookstore.dto.book.BookResponseDto;
 import bookstore.entity.book.Book;
 import org.mapstruct.InjectionStrategy;
 import org.mapstruct.Mapper;
 import org.mapstruct.MappingTarget;
 import org.mapstruct.NullValueCheckStrategy;
 import org.mapstruct.ReportingPolicy;
 
-@Mapper(componentModel = "spring",
+@Mapper(
+        componentModel = "spring",
         injectionStrategy = InjectionStrategy.CONSTRUCTOR,
         nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS,
         unmappedTargetPolicy = ReportingPolicy.IGNORE,
         implementationPackage = "<PACKAGE_NAME>.impl"
 )
 public interface BookMapper {
-    BookResponseDto toResponseDto(Book book);
+    BookResponseDto toDto(Book book);
 
     Book toModel(BookRequestDto bookRequestDto);
```

Comment:
Modify the BookMapper class to have such methods:
BookDto toDto(Book book);
Book toEntity(CreateBookRequestDto bookDto);
BookDtoWithoutCategoryIds toDtoWithoutCategories(Book book); (HINT: BookDtoWithoutCategoryIds class could be used as a response class for @GetMapping("/{id}/books") endpoint)
@AfterMapping default void setCategoryIds(@MappingTarget BookDto bookDto, Book book)